WooCommerce/WooCommerceUITests/README.md

```diff
@@ -63,9 +63,12 @@ The following flows are covered/planned to be covered by UI tests. Tests that ar
     - [ ] New order results in a push notification
     - [ ] Orders push notification opens the correct order
     - [ ] Reviews push notification opens the correct review
-7. Settings
+7. Universal Links and Deeplinks
+    - [x] Universal Link to Payments screen
```

Comment:
I decided to keep this scope to only Payments, will add the Orders universal link in a separate PR